test/metabase/pulse/render/body_test.clj

```diff
@@ -442,59 +447,35 @@
 (deftest render-area-graph-test
   (testing "Render an area graph with non-nil values for the x and y axis"
     (is (has-inline-image?
-          (render-area-graph {:cols default-columns
-                              :rows [[10.0 1] [5.0 10] [2.50 20] [1.25 30]]}))))
+         (render-area-graph {:cols         default-columns
+                             :rows         [[10.0 1] [5.0 10] [2.50 20] [1.25 30]]
+                             :viz-settings {}}))))
   (testing "Render a stacked area graph"
     (is (has-inline-image?
-          (render-stack-area-graph {:cols default-multi-columns
-                                    :rows [[10.0 1 1231 1] [5.0 10 nil 111] [2.50 20 11 1] [1.25 nil 1231 11]]}))))
+         (render-stack-area-graph {:cols         default-multi-columns
+                                   :rows         [[10.0 1 1231 1] [5.0 10 nil 111] [2.50 20 11 1] [1.25 nil 1231 11]]
+                                   :viz-settings {}}))))
   (testing "Check to make sure we allow nil values for the y-axis"
     (is (has-inline-image?
-          (render-area-graph {:cols default-columns
-                              :rows [[10.0 1] [5.0 10] [2.50 20] [1.25 nil]]}))))
+         (render-area-graph {:cols         default-columns
+                             :rows         [[10.0 1] [5.0 10] [2.50 20] [1.25 nil]]
+                             :viz-settings {}}))))
   (testing "Check to make sure we allow nil values for the y-axis"
     (is (has-inline-image?
-          (render-area-graph {:cols default-columns
-                              :rows [[10.0 1] [5.0 10] [2.50 20] [nil 30]]}))))
+         (render-area-graph {:cols         default-columns
+                             :rows         [[10.0 1] [5.0 10] [2.50 20] [nil 30]]
+                             :viz-settings {}}))))
   (testing "Check to make sure we allow nil values for both x and y on different rows"
     (is (has-inline-image?
-          (render-area-graph {:cols default-columns
-                              :rows [[10.0 1] [5.0 10] [nil 20] [1.25 nil]]}))))
+         (render-area-graph {:cols         default-columns
+                             :rows         [[10.0 1] [5.0 10] [nil 20] [1.25 nil]]
+                             :viz-settings {}}))))
   (testing "Check multiseries in one card but without explicit combo"
     (is (has-inline-image?
-          (render-multiseries-area-graph
-            {:cols default-multi-columns
-             :rows [[10.0 1 1231 1] [5.0 10 nil 111] [2.50 20 11 1] [1.25 nil 1231 11]]})))))
-
-(deftest series-with-custom-names-test
```

Comment:
These tests aren't valid anymore since I've moved the logic to calculate column/metric custom names to FE and remove `:name` from BE's data that's sent to FE static viz entirely.

@adam-james-v you added these tests. What do you think if I removed them since they're all covered by these FE unit tests instead:
https://github.com/metabase/metabase/blob/681f3bc2fa4aaff89a031d86ba60eab64c13319e/frontend/src/metabase/static-viz/components/LineAreaBarChart/utils/series.unit.spec.ts#L624-L1030